Prometheus.Client.HttpClient.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.7.34003.232
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Prometheus.Client.HttpClient", "src\Prometheus.Client.HttpClient\Prometheus.Client.HttpClient.csproj", "{C4504E36-2F22-4BAC-8772-2BF1AC960537}"
```

Comment:
I prefer single folder `src` when just one project:
`src\Prometheus.Client.HttpClient\Prometheus.Client.HttpClient.csproj` -> `src\Prometheus.Client.HttpClient.csproj"`

Reply:
@phnx47 
I personally prefer to keep project in it's own folder so that if in future we need to add any new project, it won't create a mess.

Reply:
You can create a folder late but I believe this repo is single project.